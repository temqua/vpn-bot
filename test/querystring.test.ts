import { describe, expect, test } from "@jest/globals";
import querystring from 'node:querystring';

describe("Utils testing", () => {
  test("query string test", () => {
    const qs = "username=artem&email=art_nazarov@mail.ru"
    const parsed = querystring.decode(qs)
    console.log('parsed :>> ', parsed);
    expect(parsed).toMatchSnapshot();
  });
});
