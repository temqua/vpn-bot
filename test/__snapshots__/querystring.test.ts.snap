// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Utils testing query string test 1`] = `
{
  "email": "art_nazarov@mail.ru",
  "username": "artem",
}
`;
